src/goto-programs/goto_contractor.cpp

```diff
@@ -933,9 +944,20 @@ void interval_analysis_ibex_contractor::apply_contractor()
   auto t_0 = std::chrono::steady_clock::now();
   auto X = map.create_interval_vector();
   auto c_out = contractor.get_outer();
+  auto Y = X;
+  int i = 50;
 
   c_out->contract(X);
 
+  //Find a fixed point in 5 or fewer iterations
+  while (Y != X && i >= 0)
```

Comment:
1. It looks to me like you're doing 50, not 5, iterations.
2. Where do the 50 come from and why has Ibex been crashing? We should file a bug with them or find out whether it's on our side.

Reply:
1- It should be 5. I'll change it. It is from Principles of Abstract Interpretation Book by Patrick Cousot. 
2- I think it's from their side. the crash happens when I try to access the function in the CtcFixedPoint to dump it.